<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_Calculate">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>320</y>
      <width>281</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Рассчитать</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_input">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>321</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:700; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:5px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:100%; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'Helvetica Neue','Helvetica','Roboto','Arial','sans-serif'; font-weight:400; color:#222222; background-color:#ffffff;&quot;&gt;В четверг четвертого числа в четыре с четвертью часа лигурийский регулировщик регулировал в Лигурии, но тридцать три корабля лавировали, лавировали, да так и не вылавировали.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:5px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:100%; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'Helvetica Neue','Helvetica','Roboto','Arial','sans-serif'; font-weight:400; color:#222222; background-color:#ffffff;&quot;&gt;И потом протокол про протокол протоколом запротоколировал, как интервьюером интервьюируемый лигурийский регулировщик речисто, да не чисто рапортовал, да так зарапортовался про размокропогодившуюся погоду что, дабы инцидент не стал претендентом на судебный прецедент, лигурийский регулировщик акклиматизировался в неконституционном Константинополе, где хохлатые хохотушки хохотом хохотали и кричали турке, который начерно обкурен трубкой: &amp;quot;Не кури, турка трубку, купи лучше кипу пик, лучше пик кипу купи, а то придет бомбардир из Бранденбурга – бомбами забомбардирует за то, что некто чернорылый у него полдвора рылом изрыл вырыл и подрыл.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:5px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:100%; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'Helvetica Neue','Helvetica','Roboto','Arial','sans-serif'; font-weight:400; color:#222222; background-color:#ffffff;&quot;&gt;Но на самом деле турка не был в деле, да и Клара-краля в то время кралась к ларю, пока Карл у Клары крал кораллы, за что Клара у Карла украла кларнет, а потом на дворе дёготниковой вдовы Варвары два этих вора дрова воровали; но грех – не смех – не уложить в орех: о Кларе с Карлом во мраке все раки шумели в драке, – вот и не до бомбардира ворам было, но и не до деготниковой вдовы, и не до деготниковых детей.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:5px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:100%; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'Helvetica Neue','Helvetica','Roboto','Arial','sans-serif'; font-weight:400; color:#222222; background-color:#ffffff;&quot;&gt;Зато рассердившаяся вдова убрала в сарай дрова: раз дрова, два дрова, три дрова – не вместились все дрова, и два дровосека, два дровокола, два дроворуба для расчувствовавшейся Варвары выдворили дрова вширь двора обратно на дровяной двор, где цапля чахла, цапля сохла, цапля сдохла.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:5px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:100%; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'Helvetica Neue','Helvetica','Roboto','Arial','sans-serif'; font-weight:400; color:#222222; background-color:#ffffff;&quot;&gt;Цыплёнок же цапли цепко цеплялся за цепь.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:5px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:100%; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'Helvetica Neue','Helvetica','Roboto','Arial','sans-serif'; font-weight:400; color:#222222; background-color:#ffffff;&quot;&gt;Молодец против овец, а против молодца сам овца, которой носит Сеня сено в сани, потом везет Сенька Соньку с Санькой на санках: санки – скок, Сеньку – в бок, Соньку – в лоб, все – в сугроб, а оттуда только шапкой шишки сшиб, затем по шоссе Саша пошёл, Сашу на шоссе Саша нашёл; Сонька же – Сашкина подружка шла по шоссе и сосала сушку, да притом у Соньки-вертушки во рту ещё и три ватрушки – аккурат в медовик, но ей не до медовика – Сонька и с ватрушками во рту пономаря перепономарит, перевыпономарит: жужжит, как жужелица, жужжит, да кружится: была у Фрола – Фролу на Лавра наврала, пойдет к Лавру на Фрола Лавру наврет, что – вахмистр с вахмистршей, ротмистр с ротмистршей, у ужа – ужата, у ежа – ежата, а у него высокопоставленный гость унес трость, и вскоре опять пять ребят съели пять опят с полчетвертью четверика чечевицы без червоточины, да тысячу шестьсот шестьдесят шесть пирогов с творогом из сыворотки из-под простокваши, – о всем о том около кола колокола звоном раззванивали, да так, что даже Константин – зальцбуржский бссперспективняк из-под бронетранспортера констатировал: как все колокола не переколоколовать, не перевыколоколовать, так и всех скороговорок не перескороговорить, не перевыскороговорить; но попытка – не пытка.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_output">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>60</y>
      <width>321</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:700; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:400;&quot;&gt;Результат тут&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>250</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>531</y>
      <width>671</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>P.S. Программа находит наибольшие слова, количество знаков, и количество букв в тексте</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ВВОД</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ВЫВОД</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
